tests/src/model/pages/images-page.ts

```diff
@@ -89,7 +89,7 @@ export class ImagesPage extends PodmanDesktopPage {
     }
   }
 
-  protected async imageExists(name: string): Promise<boolean> {
+  async imageExists(name: string): Promise<boolean> {
```

Comment:
please do not call this method, it was flagged as protected as it does not wait
`waitForImageExists` has to be used instead